### PR TITLE
{chem}[intel/2018b] PLUMED v2.5.1 w/ PathCV

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3126.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/2caaf4c5ad8b672b2fd14326b0923f26 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2446.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/57989c2ba77c9b3b4ba72022ae37572c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2003.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/347a9796087fe573bbad654f8486a0fe for a full test report.

File: easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.1-intel-2018b-PathCV.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': 'True'}
+
+source_urls = [
+    'https://github.com/plumed/plumed2/releases/download/v%(version)s/',
+    'http://www.acmm.nl/ensing/software/',
+]
+sources = [SOURCE_TGZ]
+patches = [('PathCV.cpp', 'src/function/')]
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.1-intel-2018b-PathCV.eb]: Where does PathCV.cpp come from?

Comment (boegel) [reply easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.1-intel-2018b-PathCV.eb]: @akesandgren http://www.acmm.nl/ensing/software/

Comment (akesandgren) [reply easybuild/easyconfigs/p/PLUMED/PLUMED-2.5.1-intel-2018b-PathCV.eb]: Ah, didn't know patches could be downloaded too. (Or have forgotten...)

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**